CMSgov/mint-app
# Easi 2622/existing model to user table

# EASI-2622

## Changes and Description

- Migrate existing model to the user table
     - NOTE this didn't follow the exact migration as the other tables. Notably, we know the state of the created by field, so it is simplified to just set it as the system account
     - It should also be noted that this field doesn't implement base struct, so the user account fields had to be implemented in the model itself



## How to test this change
1. Bring up the app
2. Navigate to Plan General Characteristics
3. Select this plan resembles an existing model
4. Select some existing models
5. Verify that you can save the application


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
